openshift/templates/event-processor/event-processor-deploy.yaml

```diff
@@ -48,11 +50,17 @@ objects:
       labels:
         name: ${NAME}${SUFFIX}
         app: ${APP_NAME}${SUFFIX}
+        app.kubernetes.io/part-of: ${APP_NAME}${SUFFIX}
         app-group: ${APP_GROUP}
         role: ${ROLE}
         env: ${TAG_NAME}
       annotations:
-        description: Defines how to deploy the container.
+        description: Defines how to deploy the application server
+        app.openshift.io/connects-to: >-
+          [{"apiVersion":"apps.openshift.io/v1","kind":"DeploymentConfig","name":"${MARA_DB_HOST}"},
+          {"apiVersion":"apps.openshift.io/v1","kind":"DeploymentConfig","name":"${EVENT_PROC_DB_HOST}"},
+          {"apiVersion":"apps.openshift.io/v1","kind":"DeploymentConfig","name":"${BC_REG_DB_HOST}"},
```

Comment:
Why do these components not use the name+suffix convention?

Reply:
not sure, I assume it's so that they can connect to db hosts that aren't within the same application. They didn't use the name+suffix convention when I started working on the project so i figured there must have been a reason for it

Reply:
I think we should update this to use the name/suffix pattern so it is consistent, rather than having a one-off configuration alongside the others.